README.md

```diff
@@ -5,8 +5,18 @@ This page is available as an easy-to-read website at [https://ebookfoundation.gi
 
 <div align="center" markdown="1">
 
-[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)
-[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+[![Awesome](https://cdn.rawgit.com/sindresorhus/awesome/d7305f38d29fed78fa85652e3a63e154dd8e8829/media/badge.svg)](https://github.com/sindresorhus/awesome)&#160;&#160;
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)&#160;&#160;
+[![Contributor Covenant 1.3](https://img.shields.io/badge/Contributor%20Covenant-1.3-4baaaa.svg)](code_of_conduct.md)&#160;&#160;
+[![Website: https://ebookfoundation.github.io/free-programming-books/](https://img.shields.io/website?down_color=red&down_message=down&up_color=green&up_message=up&url=https%3A%2F%2Febookfoundation.github.io%2Ffree-programming-books%2F&label=Website&logo=internet%20explorer&logoColor=whitesmoke&style=flat)](https://ebookfoundation.github.io/free-programming-books/)  
```

Comment:
Can we change the logo from internet explorer.
I don't have any better suggestions really, only maybe a firefox or google chrome icon.

Reply:
Internet explorer is planned to the deprecated on [15th June 2022](https://docs.microsoft.com/lifecycle/announcements/internet-explorer-11-end-of-support)

Reply:
Done at d141e975ce513b566284f9c20d230d811f9ee7cf